bazel/fuzzit_wrapper.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+# run fuzzing regression or upload to Fuzzit for long running fuzzing job depending on whether FUZZIT_API_KEY is set
+
+FUZZIT="${TEST_SRCDIR}/fuzzit_linux/fuzzit"
+
+FUZZER_BINARY=$1
+FUZZIT_TARGET_NAME="$(basename $1 | sed -e s/_fuzz_test_with_libfuzzer$// -e s/_/-/g)"
+
+if [[ ! -z "${FUZZIT_API_KEY}" ]]; then
```

Comment:
I think the previous condition was this one https://github.com/envoyproxy/envoy/pull/8835/files#diff-abef39f4a7cbfbedd8850dcd8fbcc792L145

as we wanted the fuzzing to run on push to master and not on pull-requests. This is becuase fuzzing is essentially a long running job that only stops when a new crash is found or when new code is pushed and then the new fuzzers starts and the old one stops (keeping the corpus between runs).

Reply:
@yevgenypats yes this is intended, I merged two jobs into one. In our CI setup, PR runs doesn’t have FUZZIT_API_KEY set so on master this run fuzzing + local-regression, and on PR run this only run local-regression.